.github/workflows/ruby_build.yml

```diff
@@ -1,33 +0,0 @@
-name: Ruby Build
```

Comment:
Just like with SS, could we restore this with the updated release action please.

Reply:
👍 I've done this but it's displayed separately above because of the correct indentation in the new file